contents/page-sections/faq/with-image/index.tsx

```diff
@@ -11,44 +11,15 @@ import FAQ from "./faq.svg"
 const WithImage = () => {
   return (
     <Flex
-      _container={[
-        {
-          maxW: "900px",
-          css: {
-            flexDir: "column",
-          },
-        },
-      ]}
+      flexDir={{ base: "row", md: "column" }}
```

Comment:
Isn't it equivalent to this?

```suggestion
      flexDir={{ lg: "column" }}
```